pkg/agent/multicast/mcast_discovery.go

```diff
@@ -127,7 +127,7 @@ func (s *IGMPSnooper) processPacketIn(pktIn *ofctrl.PacketIn) error {
 		fallthrough
 	case protocol.IGMPv2Report:
 		mgroup := igmp.(*protocol.IGMPv1or2).GroupAddress
-		klog.InfoS("Received IGMPv1or2 Report message", "group", mgroup.String(), "interface", iface.PodName)
+		klog.InfoS("Received IGMPv1or2 Report message", "group", mgroup.String(), "interface", iface.InterfaceName)
```

Comment:
maybe it is easier to keep pod name as well for debuging?

Reply:
I change it to InterfaceName because I thought after we support packetIn IGMP from tunnel port which is possibly used with encap mode, PodName field is possible empty.

Reply:
added.